packages/terra-form-select/tests/jest/MenuUtil.test.jsx

```diff
@@ -302,7 +302,7 @@ describe('MenuUtil', () => {
         <OptGroup label="2">{options.slice(2, 5)}</OptGroup>,
       ];
 
-      expect(MenuUtil.findNext(group, '5')).toEqual('5');
+      expect(MenuUtil.findNext(group, '5')).toEqual(null);
```

Comment:
findNext for last element should return 1st element not null.

Reply:
Changed as suggested.